docs/version0.24.md

```diff
@@ -32,6 +32,7 @@ The following new features and notable changes since v 0.23.0 are included in th
 - [`-XX:[+|-]PortableSharedCache` option behavior update](#-xx-portablesharedcache-option-behavior-update)
 - [![Start of content that applies to AIX Java 15+](cr/java15plus.png) `-XX:[+|-]ShareAnonymousClasses` option behavior update](#-xx-shareanonymousclasses-option-behavior-update)
 - [Additional parameters for `jcmd Dump` commands](#additional-parameters-for-jcmd-dump-commands)
+- [Update to compile Windows Java 8 with VS2013](#update-to-compile-windows-java-8-with-vs2013)
```

Comment:
As this relates to builds, I would be more inclined to include this information in the "Binaries and supported environments" section of this topic.  Please can you just lift the sentence into there, before the final "For more information, ...". Unless of course we are talking only about OpenJ9 builds using this compiler level and not AdoptOpenJDK?

Reply:
It affects AdoptOpenJDK as well. I've made a change, not sure if it's what you had in mind.